src/types/video.ts

```diff
@@ -80,7 +80,7 @@ export enum SelectedTrackType {
 
 export type SelectedTrack = {
   type: SelectedTrackType;
-  value?: string | number;
```

Comment:
Value can be a number in the code ... it is used when we select by index.
Is it problematic for codegen ?

Reply:
Ah, all types in `src/types/video.ts` are for JS layer's prop types.
The reason I removed `number` is I did want to sync with codegen types of SelectedTrack.
Such as this https://github.com/react-native-video/react-native-video/pull/3487/files/d1a858a590339b48d5430fbe8b8769f8f1921bd6#diff-70773b05560f0897b24b4baa0ed114158fd50e075c0c76de43cc30b90043eb80R85

![스크린샷 2024-01-31 오전 10 19 48](https://github.com/react-native-video/react-native-video/assets/36250002/eb6ea7a7-75cb-48ef-90ea-3aef33308600)

I think Codegen do not allow mixed types.

Reply:
Maybe we have to change docs content of this types.
I am not sure what is the best option. But codegen do not allow mixed types.
So maybe we have to change docs?
Let me know your opinion @freeboub

Reply:
ok, I can propose you this change:
- keep current behavior fo external API
- move the value to string only in native bindings (then string To Int in native code)
- do the wrapping in Video.js to convert the int to string

This is the ideal change. but is you really prefer keep it as string only (even in external API), just notice this is a breaking change. (but you need anyway to change the native code)

Reply:
okay I will have a check.
Thanks for the suggestions!

Reply:
@freeboub hello, sorry for the late reply.
I've implemented your suggestions on this commit https://github.com/react-native-video/react-native-video/pull/3487/commits/2ef456a23206201b444c50bd316d42677048dca0

Can you have a look?
If I didn't catch your suggestions, let me know.